### PR TITLE
Add styled-components

### PR DESCRIPTION
**Pull request by bgribz**
- Adds the `styled-components` package
- Tidies up the Header a smidgen

### CODE REVIEW

File: web-app/src/components/Header/styled.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const HeaderContainer = styled.div`
+  background-color: #0eb5d1;
```
Comment (ghost) [inline web-app/src/components/Header/styled.js]: It'd be cool if you could make variables for colours.

File: web-app/src/styled.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { injectGlobal } from 'styled-components';
+
+injectGlobal`
+  body {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
```
Comment (ghost) [inline web-app/src/styled.js]: I know fonts etc haven't been sorted yet but if we could just create a stub font variable then change it as we move along.

File: web-app/src/styled.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { injectGlobal } from 'styled-components';
```
Comment (ghost) [inline web-app/src/styled.js:1]: Are we tied to this injectGlobal term? If we wanted to theme this it'd need changed. Maybe unosquareGlobal or something. Just make it more flexible.

File: web-app/src/components/Header/styled.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 25px;
+  color: white;
```
Comment (ghost) [inline web-app/src/components/Header/styled.js]: Variable here also

File: web-app/src/components/Header/index.jsx [hunk 5]
```diff
@@ -1,75 +1,40 @@
...
-import 'react-bootstrap';
 import LoginService from '../../pages/Login/LoginService';
-import styles from './style.css';
 import roles from '../../utilities/roles';
+import { HeaderContainer, HeaderItem } from './styled';
 
 class Header extends React.Component {
     constructor(props){
         super(props);
-        this.handleLogout = this.handleLogout.bind(this);
```
Comment (ghost) [inline web-app/src/components/Header/index.jsx]: Nice

File: web-app/src/components/Header/styled.js (hunk 1 above)
Comment (bgribz) [reply web-app/src/components/Header/styled.js]: We can! I'll edit this PR to use styled-components' theme feature :+1:

File: web-app/src/styled.js (hunk 3 above)
Comment (bgribz) [reply web-app/src/styled.js:1]: We are but it's just styled-components' way of injecting global CSS styles without tying them to any one component.

**Review by BrownMark — APPROVED** [review]
Looks fine Barry, you just need to merge with develop.

**Pull request merged**